### PR TITLE
feat/VEGA-5307 added new navigation

### PR DESCRIPTION
**Pull request by SergeyKatugin**
## Problem statement
https://artcpt.atlassian.net/browse/VEGA-5308

## Solution details
Обновление навигации в shell

## Type
- [x] Feature
- [ ] Bug Fix
- [x] Refactoring
- [x] Test
- [ ] Other

## Quality control
- [x] PR: Based on a correct branch
- [x] PR: Head branch is rebased on the base branch
- [x] PR: Assignee chosen and necessary labels are set
- [x] PR: Current form is filled out (where it makes sense)
- [x] PR: Diff checked for irrelevant changes
- [x] PR: Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specs
- [x] PR: Checked for spelling, grammar and typos
- [x] Docs: All the important changes and features are described
- [ ] Tests: New unit tests developed
- [ ] Tests: New E2E tests developed
- [ ] Tests: Existing unit tests passed successfully
- [ ] Tests: Existing E2E tests passed successfully
- [ ] Tests: Manually tested on personal instance

## Notable statements
- [ ] Affects configuration files
- [ ] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

### CODE REVIEW

**Review by satrasmatras — APPROVED** [review]

File: src/constants/RoutesConstant.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import { MicrofrontendModule } from './MicrofrontendConstants';
```
Comment (ndrzhn) [inline src/constants/RoutesConstant.ts:1]: название файла лучше бы во множественном числе, чтобы было однородно` RoutesConstants.ts`

File: src/store/initStore.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      : [epicMiddleware];
+
+  const store = createStore(
+    rootReducer,
+    process.env.NODE_ENV === 'development'
+      ? composeWithDevTools(applyMiddleware(...middleware))
+      : applyMiddleware(...middleware),
+  );
+
+  // epicMiddleware.run(rootEpic);
```
Comment (ndrzhn) [inline src/store/initStore.ts:48]: коммент нужен?

File: src/store/shell/shellActions.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import actionCreatorFactory from 'typescript-fsa';
+
+import { MicrofrontendModule } from '../../constants/MicrofrontendConstants';
+
+const factory = actionCreatorFactory('sell');
```
Comment (ndrzhn) [inline src/store/shell/shellActions.ts:5]: `sell` - опечатка, имеется ввиду `shell`?

File: src/constants/RoutesConstant.ts (hunk 1 above)
Comment (SergeyKatugin) [reply src/constants/RoutesConstant.ts:1]: ok

File: src/store/initStore.ts (hunk 2 above)
Comment (SergeyKatugin) [reply src/store/initStore.ts:48]: да, добавлю пояснение

File: src/store/shell/shellActions.ts (hunk 3 above)
Comment (SergeyKatugin) [reply src/store/shell/shellActions.ts:5]: ага) поправлю

**Pull request merged**